Britannia/localization/english/event_localization/BTWK_events_l_english.yml

```diff
@@ -1,25 +1,50 @@
+  BTWK_frontier_event.0201.a:0 "Out with the old..."
+  BTWK_frontier_event.0201.a_tt:0 "All instances of the [special_buildings|E] #italic Hadrian's Wall#! or #italic Hadrian's Wall - Restored#! will be ruined."
+
+  BTWK_frontier_event.0202:0 "The End of Hadrian's Wall"  # Every other player
+  BTWK_frontier_event.0202_desc_header:0 "[GetScope(destroyed_hadrians_wall_char).Char.GetName] has ordered the demolition of Hadrian's Wall"
+  BTWK_frontier_event.0202_desc_header_vassal:0 "Our liege, [GetScope(destroyed_hadrians_wall_char).Char.GetName], has ordered the demolition of Hadrian's Wall."
+  BTWK_frontier_event.0202_desc_main:0 "\n\n [GetScope(destroyed_hadrians_wall_char).Char.GetHerHis] hopes that now the wall no longer serves any meaningful purpose, its stone could be repurposed for the developing settlements throughout [GetScope(destroyed_hadrians_wall_char).Char.GetHerHis] realm."
+  BTWK_frontier_event.0202_desc_imperial_like_destroyer:0 "[GetScope(destroyed_hadrians_wall_char).Char.GetFirstName] has done this despite [GetScope(destroyed_hadrians_wall_char).Char.GetHerHis] his heritage, from which is intrinsically linked to the momument. For this act, many of his kinsmen have been severely angered."
+  BTWK_frontier_event.0202_desc_else_destroyer:0 "Having no ancient ties to the land, and the wall which sits atop it, [GetScope(destroyed_hadrians_wall_char).Char.GetFirstNamePossessiveNoTooltipRegnal] decision was  tenable. The act has angered the likes of the Britons and of the Latins, who have ancient ties to the monument."
+  BTWK_frontier_event.0202.a_imperial_like:0 "A terrible day for Britannia"
+  BTWK_frontier_event.0202.a_other_0 "Interesting news" 
+  BTWK_frontier_event.0202.a_other_vassal:0 "For the good of the realm!"
+
+  BTWK_frontier_event.0203:0 "Hadrian's Wall Restored"  # ROOT of restoration
+  BTWK_frontier_event.0203_desc:0 "The conclusion of the renovations in [GetGlobalVar('last_old_hadrians_wall_province').MakeScope.Province.GetProvince] has marked the complete restoration of Hadrian's Wall. While the gesture was ultimately symbolic as a reminder of the Imperial roots all Britons share, the wall is now operable as the defensive frontline as it once was, should the need ever arise again. \n\nThough the effort to rebuild has taken many years, gold and manpower to complete, as the overseer of the final stones being set into place, you are the one the poems shall remember the most. Many among the isle are beginning to praise you as 'Hadrianus', as a successor to the great Emperor Hadrian himself. Perhaps this new reputation might aid your bids to become High King of Britannia, should you wish it..."
```

Comment:
While the gesture was ultimately symbolic as a reminder of the Imperial roots all Britons share -> While the gesture was ultimately symbolic as the wall no longer marks the northern frontier of Britannia
Perhaps this new reputation might aid your bids to become High King of Britannia, should you wish it... -> Perhaps this new reputation might be the making of a new High King... (probably need to make this conditional in case the ruler is already High King, or if Britannia as already collapsed)

BTW is this event specifically meant for a British restoration of Hadrian's Wall, or one by anyone?

Reply:
This event is the ROOT who triggers the restoration. That can only be Imperial, British or Romanised Germanic per our previous discussions. Britons in this case means residents of the isle but I can see how easily that could be confused with the specific culture. 

Will make sure to consider the conditional ones though. I assume by "if Britannia has collapsed" you're referring to the 'Fall of Britannia' event chain?

Reply:
I see, I was definitely confused by the use of "Britons" there.

And yes, it would be the Fall of Britannia event (which is meant to permanently disables the High King mechanic)